src/lxml/includes/etree_defs.h

```diff
@@ -247,6 +247,12 @@ long _ftol2( double dblSource ) { return _ftol( dblSource ); }
 #define _isString(obj)   (PyUnicode_Check(obj) || PyBytes_Check(obj))
 #endif
 
+#if PY_VERSION_HEX >= 0x03060000
+#define PY_FSPath(obj) (PyOS_FSPath(obj))
+#else
+#define PY_FSPath(obj) (NULL)
+#endif
```

Comment:
This is a public header file, so we should avoid introducing user side collisions where possible.
```suggestion
#if PY_VERSION_HEX >= 0x03060000
#define lxml_PyOS_FSPath(obj) (PyOS_FSPath(obj))
#else
#define lxml_PyOS_FSPath(obj) (NULL)
#endif
```